heareval/evaluation/task_evaluation.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+"""
+Contains a set of various evaluation scores. Runs evaluation on the test predictions
+for predictions on a HEAR task.
+Would it make sense to move all the score functions over to a file called scores?
```

Comment:
Let's update this docstring, looks like this was done here

Reply:
Sure will do. We might end up removing this file or not having this as a separate step from prediction.